queue_job/models/queue_job.py

```diff
@@ -231,6 +232,9 @@ def _compute_graph_jobs_count(self):
 
     @api.model_create_multi
     def create(self, vals_list):
+        test_mode = getattr(threading.current_thread(), 'testing', False)
+        if not vals_list and test_mode:
+            return super().create(vals_list)
```

Comment:
so we loose `with_context(mail_create_nolog=True, mail_create_nosubscribe=True)`?
needed?